L1Trigger/Phase2L1ParticleFlow/plugins/L1TCorrelatorLayer1Producer.cc

```diff
@@ -488,6 +512,13 @@ void L1TCorrelatorLayer1Producer::initSectorsAndRegions(const edm::ParameterSet
       }
     }
   }
+
+  event_.board_out.clear();
+  const std::vector<edm::ParameterSet> &board_params = iConfig.getParameter<std::vector<edm::ParameterSet>>("boards");
+  event_.board_out.resize(board_params.size());
+  for (unsigned int bidx = 0; bidx < board_params.size(); bidx++) {
+    event_.board_out[bidx].region_index = board_params[bidx].getParameter<std::vector<unsigned int>>("regions");
+  }
```

Comment:
perhaps this could go in `initSectorsAndRegions`?

Reply:
mmhh this is in `initSectorsAndRegions` no?

Reply:
ah sorry, the collapsed lines in the diff confused me  ;-)